test/e2e/common/configmap.go

```diff
@@ -73,12 +72,8 @@ var _ = framework.KubeDescribe("ConfigMap", func() {
 	})
 
 	It("updates should be reflected in volume [Conformance] [Volume]", func() {
-
-		// We may have to wait or a full sync period to elapse before the
-		// Kubelet projects the update into the volume and the container picks
-		// it up. This timeout is based on the default Kubelet sync period (1
-		// minute) plus additional time for fudge factor.
-		const podLogTimeout = 300 * time.Second
+		podLogTimeout := framework.GetPodSecretUpdateTimeout(f.ClientSet)
+		mt_retry_time := fmt.Sprintf("--retry_time=%v", int(podLogTimeout.Seconds()))
```

Comment:
This is very non-go name - please use camel case. Plus I don't know what 'mt' stands for.